go.mod

```diff
@@ -4,6 +4,7 @@ go 1.12
 
 require (
 	github.com/davecgh/go-spew v1.1.1
+	github.com/gorilla/mux v1.6.3-0.20180517173623-c85619274f5d // indirect
```

Comment:
weird why it's pulling this stuff in, your change doesn't reference this.

Reply:
Tim and I fixed it :)